### ISSUE TITLE
Change request: 'view source' and 'Edit on GitHub' buttons could open in a new window

### ISSUE THREAD

**Issue by vb-benjamin-eyring** [TYPO3-Documentation/sphinx_typo3_theme#182]
Hi all, 

![Bildschirm­foto 2023-01-05 um 14 36 17](https://user-images.githubusercontent.com/113903434/210792494-d280ee04-450e-48a2-8b87-be89627d3f53.png)

I would like to propose to open the buttons 'View source' and 'Edit on GitHub' in a new window like 'How to edit' already does. From my point of view this would be the expected behaviour. 

I already had a discussion about that in slack with Lisa Wolf and I also can understand her point of view:

https://typo3.slack.com/archives/C028JEPJL/p1672925228168839

Not sure how to proceed from here - I could ask some colleagues on their opinion. Or do we have a voting tool at hand? Or do you decide in some team or forum on such ideas?

Thx,
Ben

**Comment by sypets** [comment]
My 2 c: A quite simple method is to upvote or download an issue using :thumbsup: or :thumbsdown: on the issue.

This is quite common in other open source projects (but not so common in TYPO3, but we could establish it here as well).

Since I am too lazy to open Slack and not everyone is in Slack, could you summarize the argument that were given? Please no copy pasting of screenshots.

All of the above is my personal opinion, not official.

About the proposal: I agree and have used :thumbsup:

**Comment by vb-benjamin-eyring** [comment]
Up/Downvote sounds good! As you mentioned that in general there is no democratic process for decisions - then who makes the decision? Would I have to approach anyone for such a change request? Or are the change requests scanned on a regular basis?

**Comment by DavidBruchmann** [comment]
I agree on the main proposition to open the link in a new window.  
Also the link "how to edit" should perhaps open in a new window.  
My reason is that those sites might be viewed side-by-side to get any edits in the fastest and best way resolved and that it's probably not helpful just to change the current window content to another site.

**Comment by DavidBruchmann** [comment]
Concerning the voting I object, because down-voting might tend to discourage people from engagement. While it might be helpful for routined users, especially new users see it as a negative feedback beyond a simple voting process.

**Comment by DavidBruchmann** [comment]
@linawolf you keep your opinion about the target of the buttons or you could reconsider perhaps?

**Comment by marble** [comment]
I tend to use "open in new window" for all of the three buttons and see whether there is some protest. However, I'll do some experiments on different devices first.

**Comment by vb-benjamin-eyring** [comment]
That sounds good to me. thx
